### PR TITLE
Bmakau

### PR DESCRIPTION
**Pull request by bmakau**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -4,21 +4,22 @@ spring.h2.console.enabled=true
...
+### create: creates the schema, destroying previous data.
+### create-drop: drop the schema at the end of the session.
+#spring.jpa.hibernate.ddl-auto=update
+#spring.datasource.initialSize=5
+#spring.datasource.maxAge=180000
+#spring.datasource.hikari.maximum-pool-size=5
+#spring.datasource.hikari.connection-timeout=5000
+#spring.datasource.testWhileIdle = true
+#spring.datasource.validationQuery = SELECT 1
+##spring.datasource.maxActive=5
```
Comment (ckirkwood94) [inline src/main/resources/application.properties]: Uncomment lines 8, and 17-24

File: src/main/java/com/lambdaschool/wellness/controllers/GroupController.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    }
+
+    @PostMapping("")
+    public ResponseEntity<?> addNewGroup(@Valid @RequestBody Group newGroup) throws URISyntaxException
+    {
+        newGroup = groupService.save(newGroup);
+        HttpHeaders responseHeaders = new HttpHeaders();
+        URI newUserURI = ServletUriComponentsBuilder.fromCurrentRequest().path("/groupid").buildAndExpand(newGroup.getGroupid()).toUri();
+        responseHeaders.setLocation(newUserURI);
+        return new ResponseEntity<>(responseHeaders, HttpStatus.CREATED);
```
Comment (ckirkwood94) [inline src/main/java/com/lambdaschool/wellness/controllers/GroupController.java]: Would it be beneficial to return the new group object after creating?

File: src/main/java/com/lambdaschool/wellness/controllers/CompetitionController.java [hunk 3]
```diff
@@ -13,77 +13,61 @@
...
-    public ResponseEntity<?> addNewGroup(@Valid @RequestBody Competition newGroup) throws URISyntaxException
+    public ResponseEntity<?> addNewComp(@Valid @RequestBody Competition newComp) throws URISyntaxException
     {
-        newGroup = competitionService.save(newGroup);
+        newComp = competitionService.save(newComp);
         HttpHeaders responseHeaders = new HttpHeaders();
-        URI newUserURI = ServletUriComponentsBuilder.fromCurrentRequest().path("/groupid").buildAndExpand(newGroup.getGroupid()).toUri();
+        URI newUserURI = ServletUriComponentsBuilder.fromCurrentRequest().path("/compid").buildAndExpand(newComp.getCompid()).toUri();
         responseHeaders.setLocation(newUserURI);
         return new ResponseEntity<>(responseHeaders, HttpStatus.CREATED);
```
Comment (ckirkwood94) [inline src/main/java/com/lambdaschool/wellness/controllers/CompetitionController.java]: Same with groups. Would it be better to return the competition object in the response?

**Comment by ckirkwood94** [comment]
Overall looks good! Just need to uncomment the lines in application.properties file and I'll approve

File: src/main/java/com/lambdaschool/wellness/controllers/GroupController.java (hunk 2 above)
Comment (bmakau) [reply src/main/java/com/lambdaschool/wellness/controllers/GroupController.java]: yeah it is possible

**Pull request merged**